modules/intentIqAnalyticsAdapter.js

```diff
@@ -249,14 +228,14 @@ function bidWon(args, isReportExternal) {
     return;
   }
 
-  if (
-    iiqAnalyticsAnalyticsAdapter.initOptions.lsValueInitialized &&
-            !iiqAnalyticsAnalyticsAdapter.initOptions.lsIdsInitialized
-  ) {
-    initReadLsIds();
+  const success = receivePartnerData();
+  if (success) {
```

Comment:
Should this condition be negated? If not successful, then return and skip init. 

Does this need to happen in the BidWon function, it will repeat with the same initialization data, should it be part of the Analytics adapter's `initAdapterConfig` function as part of it's enablement